$OpenBSD: patch-src_nautilus-file-operations_c,v 1.2 2018/03/29 19:58:09 ajacoutot Exp $

nautilus-file-operations.c:508: error: #pragma GCC diagnostic not allowed inside functions
nautilus-file-operations.c:509: error: #pragma GCC diagnostic not allowed inside functions
nautilus-file-operations.c:541: error: #pragma GCC diagnostic not allowed inside functions
nautilus-file-operations.c:871: error: #pragma GCC diagnostic not allowed inside functions
nautilus-file-operations.c:872: error: #pragma GCC diagnostic not allowed inside functions
nautilus-file-operations.c:904: error: #pragma GCC diagnostic not allowed inside functions

Index: src/nautilus-file-operations.c
--- src/nautilus-file-operations.c.orig
+++ src/nautilus-file-operations.c
@@ -505,8 +505,6 @@ get_link_name (const char *name,
         use_count = TRUE;
     }
 
-#pragma GCC diagnostic push
-#pragma GCC diagnostic ignored "-Wformat-nonliteral"
     if (use_count)
     {
         result = g_strdup_printf (format, count, name);
@@ -538,7 +536,6 @@ get_link_name (const char *name,
             g_free (new_name);
         }
     }
-#pragma GCC diagnostic pop
     return result;
 }
 
@@ -872,8 +869,6 @@ make_next_duplicate_name (const char *base,
         use_count = TRUE;
     }
 
-#pragma GCC diagnostic push
-#pragma GCC diagnostic ignored "-Wformat-nonliteral"
     if (use_count)
     {
         result = g_strdup_printf (format, base, count, suffix);
@@ -905,7 +900,6 @@ make_next_duplicate_name (const char *base,
             g_free (new_base);
         }
     }
-#pragma GCC diagnostic pop
 
     return result;
 }
